main.go

```diff
@@ -100,13 +103,14 @@ func main() {
 		return
 	}
 
-	if len(os.Args) < 6 {
+	if flag.NArg() != 5 {
+		log.Printf("Error: Invalid number of arguments (got %d, expected 5)\n\n", flag.NArg())
```

Comment:
This one caught me out a few times when first on-boarding with this command (as no specific error was printed).